docs/source/14_contribution/technical_steering_committee.md

```diff
@@ -0,0 +1,43 @@
+
+In this section, we detail:
+ * [Responsibilities of a maintainer](#responsibilities-of-a-maintainer)
+ * [Requirements to become a maintainer](#requirements-to-become-a-maintainer)
+ * [Application process](#application-process)
+ * [Removing a maintainer](#removing-a-maintainer)
+
+
+## Responsibilities of a maintainer
+
+| Theme | Activity |
+|---|---|
+| Product development | * Be available for at least one full day per week to help with product development<br>* Attend community meetings to discuss the project plans and roadmap<br>* Project maintenance including security, updates, CI/CD, builds and infrastructure<br>* Prioritize the work following the product roadmap to move the project forward |
+| Community management | * Make sure that ongoing PRs are moving forward at the right pace or closing them<br>* Guide the community to use the right channel:<br>  * [Github](https://github.com/quantumblacklabs/kedro/) for feature requests and bug reports<br>  * [GitHub discussions](https://github.com/quantumblacklabs/kedro/discussions) or [Discord](https://discord.gg/akJDeVaxnB), but the former is often preferable since it will be picked up by search engines |
+
+
+## Requirements to become a maintainer
+
+Just contributing does not make you a maintainer; you need to demonstrate commitment to Kedro's long-term success by
+working with existing maintainers for a period.
+
+We look for commitment markers with the following:
+
+* Being able to write solid code and collaborate with the team and community
+* Understanding the project's code base and internals
+* Doing PRs from our backlog or roadmap; maintainers need to work towards a common goal
+* Learning how the team works, including processes for testing, quality standards and code review
+* Evidence of already having started PRs and code reviews under the guidance of maintainers; including asking for help where needed
+
+
+## Application process
+
+Quarterly, existing maintainers will curate a list of contributors that have shown regular activity on the project over the prior months. From this list, maintainer candidates are selected and proposed for a vote.
+
+Following a successful vote, candidates are added to the `kedro-developers` team on the Kedro GitHub organisation
+and the `kedro-team` group on the Kedro Discord organisation.
+
+
+# Removing a Maintainer
+The TSC will also have a mechanism for removing a maintainer is no longer interested or cannot perform the maintainer duties listed above. In that case, they should volunteer for non-voting Contributor or Friends status. In extreme cases, this can also occur by a vote by the active TSC.
```

Comment:
```suggestion
The TSC will also have a mechanism for removing a maintainer who is no longer interested or cannot perform the maintainer duties listed above. In that case, they should volunteer for non-voting Contributor or Friends status. In extreme cases, this can also occur by a vote by the active TSC.
```